### PR TITLE
Removing `no_args_is_help` from CLI commands

### LINKED ISSUE #26: [BUG] Remove `no_args_is_help` from CLI
**Describe the bug**
If you use a morpheus CLI command without any arguments or options, it just prints the help instead of adding that stage

**Steps/Code to reproduce bug**
For example, this launch command will just print the help:

```bash
(morpheus)$ morpheus run pipeline-nlp \
                from-file --filename=data/pcap_dump.jsonlines \
                deserialize
Configuring Pipeline via CLI
Usage: morpheus run pipeline-nlp deserialize [OPTIONS]

Options:
  --help  Show this message and exit.
```

**Expected behavior**
The pipeline should run with the specified commands

### PR DESCRIPTION
**Pull request by mdemoret-nv**
Closes #26

### CODE REVIEW

File: morpheus/cli.py [hunk 1]
```diff
@@ -125,8 +124,6 @@ def new_func(*args, **kwargs):
 
             kwargs = _without_empty_args(kwargs)
 
-            CppConfig.set_should_use_cpp(kwargs.pop("use_cpp", CppConfig.get_should_use_cpp()))
```
Comment (cwharris) [inline morpheus/cli.py:128]: This seems unrelated. Is this being removed intentionally, and why?

File: morpheus/cli.py [hunk 2]
```diff
@@ -306,7 +310,11 @@ def run(ctx: click.Context, **kwargs):
     pass
 
 
-@click.group(chain=True, short_help="Run the inference pipeline with a NLP model", cls=AliasedGroup, **command_kwargs)
+@click.group(chain=True,
+             short_help="Run the inference pipeline with a NLP model",
+             no_args_is_help=True,
```
Comment (cwharris) [inline morpheus/cli.py:315]: does "args" include the stages, such as `from-file`, or just the `--` flags? If it's just the flags, then this might be problematic, since in the issue we use
```
morpheus run pipeline-nlp \
             from-file --filename=data/pcap_dump.jsonlines \
             deserialize
```
where pipeline-nlp has no flags. But I assume `from-file` counts as an "arg", and this is fine.

**Review by cwharris — CHANGES_REQUESTED** [review]

**Review by dagardner-nv — APPROVED** [review]
LGTM

File: morpheus/cli.py (hunk 2 above)
Comment (mdemoret-nv) [reply morpheus/cli.py:316]: This only applies to the last command in the chain. So this would not trigger the help for `pipeline-nlp`:
```
morpheus run pipeline-nlp \
             from-file --filename=data/pcap_dump.jsonlines \
             deserialize
```
But this would:
```
morpheus run pipeline-nlp
```
Since `morpheus run pipeline-nlp` would be an invalid pipeline with no commands, it would be better to just print the help info. The `no_args_is_help` option should be enabled for all `@cli.group` decorators, especially ones with `@cli.group(chain=True)`

File: morpheus/cli.py (hunk 1 above)
Comment (mdemoret-nv) [reply morpheus/cli.py:128]: This was a merge error which I tried to fix this this PR at the same time but I forgot to add it back into the `run` command (Good catch because I had the fix wrong).

This value should only be set in the `run` function which actually has the `--use_cpp` option. Setting it in `prepare_command` will just set the same value over and over again.

**Review by cwharris — APPROVED** [review]
